.github/workflows/semantic-pr-title.yml

```
@@ -8,6 +8,10 @@ on:
       - synchronize
   workflow_dispatch:
 
+concurrency:
+  group: semantic-pr-title-${{ github.head_ref }}
```

```suggestion
  group: ${{ github.workflow }}-${{ github.head_ref }}
```

Exactly the same, but more reusable.